ui/app/adapters/secret-engine.js

```diff
@@ -135,11 +135,7 @@ export default ApplicationAdapter.extend({
 
   saveZeroAddressConfig(store, type, snapshot) {
     const path = encodePath(snapshot.id);
-    const roles = store
-      .peekAll('role-ssh')
-      .filterBy('zeroAddress')
-      .mapBy('id')
-      .join(',');
+    const roles = store.peekAll('role-ssh').filterBy('zeroAddress').mapBy('id').join(',');
```

Comment:
again, is this a linting thing? What's the rule change here?

Reply:
I checked and the `printWidth` did not change between versions so I'm not sure why this changed. I'm going to double check that the `prettier-eslint` cli is using the project config.